wizeline/tea
# Feat - Install Style Guidist

## Purpose
Install styleguidist node package so that we can start adding and documenting UI components.

## Issue
https://github.com/wizeline/phoenix/issues/8

## Main files of interest
`styleguide.config.js`

## Steps to check it works
1. `yarn install`
2. `npm run styleguide`
3. Open `localhost:6060` in your favourite browser
4. Check that the page loads and looks like the screenshot below

## Screenshots
![screen shot 2018-01-03 at 3 00 20 pm](https://user-images.githubusercontent.com/10840350/34539672-0f32401c-f097-11e7-9a99-69cc610410a0.png)
